newIDE/app/src/EventsSheet/ToolbarCommands.js

```diff
@@ -38,6 +39,10 @@ const ToolbarCommands = (props: Props) => {
     handler: props.onAddCommentEvent,
   });
 
+  useCommand('TOGGLE_EVENT_DISABLED', true, {
```

Comment:
here, instead of `true`, we should know when the command can be used or not.
I'd suggest passing down a prop `canToggleEventDisabled` coming from the component above, corresponding to `hasEventSelected(this.state.selection)`